src/components/Shared/AmountInput/AmountInput.tsx

```diff
@@ -1,46 +1,91 @@
-import type { FC } from "react";
```

Comment:
Should this have a test or is it covered by the other updated tests?

Reply:
Will be done in #306 :+1: